widget/embedded/src/pages/LiquiditySourcePage.tsx

```diff
@@ -133,7 +146,13 @@ export function LiquiditySourcePage({
 
```

Comment:
Please, use i18n for text field placeholder

Reply:
fixed